examples/empty-migration.js

```diff
@@ -0,0 +1 @@
+module.exports = function () {};
```

Comment:
Maybe we should stop putting all these e2e test related files into the examples directory.  
This is definitely not a useful example for anyone ;)

Reply:
Was thinking the same, but put them there to not start a new "structure paradigm" for such a simple change.